Dockerfile

```diff
@@ -1,26 +1,31 @@
-FROM rhel7
+# Using playbook2image as a base
+# See https://github.com/aweiteka/playbook2image for details on the image
+# including documentation for the settings/env vars referenced below
+FROM docker.io/aweiteka/playbook2image:latest
 
-MAINTAINER Troy Dawson <tdawson@redhat.com>
+LABEL name="openshift-ansible" \
+      summary="OpenShift's installation and configuration tool" \
+      description="A containerized openshift-ansible image to let you run playbooks to install, upgrade, maintain, chek, etc. an OpenShift cluster" \
```

Comment:
nit: typo s/chek/check

I'd probably remove the "etc", for it doesn't add information for users/customers.